### ISSUE TITLE
Add the ability to generate test targets

### ISSUE THREAD

**Issue by JesseTatasciore** [Evertz/bzlgen#9]
When there are test files present, it would be useful to generate the targets necessary to run those tests

**Comment by mattem** [comment]
Does the functionality exist in our previous version of gen internally? I think it generated the `ts_web_test` and `ng_web_test` macros.

Do we still want that here or should it gen a `ts_library` and the`karma_web_test_suite`?

**Comment by JesseTatasciore** [comment]
Yes our previous internal version generated for the macros. I would be inclined to say macro. Allows for mass customization of targets / what if someone wants an additional target for whatever reason. Leaves the implementation out of our hands

**Comment by mattem** [comment]
Seems reasonable, we should look to add `ng_web_test` and `ts_web_test` to the index.bzl here. I quite like those as macros anyway.

**Comment by Toxicable** [comment]
So for our repository we use `jasmine_node_test` for all our unit testing, this includes both nodejs and browser targets.

Might be worth making it configurable?
